confidence/models.py

```diff
@@ -46,6 +46,29 @@ def unwrap(source: typing.Any) -> typing.Any:
     return source
 
 
+def union(*sources: typing.Mapping[str, typing.Any], missing: typing.Any = None) -> 'Configuration':
```

Comment:
Not sure the name `union` is great. Would `combine` be better? (`merge` could work, but would greatly complicate the existence of `merge` in `utils.py`). Any other ideas maybe?

Reply:
Discussed offline, `merge` would be the best name, `combine` is second choice. The real choice is renaming `confidence.utils.merge`...

Reply:
Original `merge` in utils has been renamed to `merge_into` (as it does an in-place update), `union` in models is renamed to `merge (what we kind of wanted), and `confidence.utils.merge` is now an explicitly deprecated alias.